Approved/implicitly-scoped-qubit-allocation.md

```diff
@@ -0,0 +1,347 @@
+---
+title: Implicitly-scoped qubit allocation
+description: Provides more convenient syntax for allocating qubits.
+author: Sarah Marshall
+date: September 28, 2020
+---
+
+# Proposal
+
+1. Add new `use` and `borrow` statements to allocate qubits that are valid until the end of the current block.
+2. Add new scope statement that does not require a preceding keyword, and is executed exactly once; equivalent to `if (true) { }`.
```

Comment:
I am a bit skeptical about the deprecation of the block-version in favor of a scope statement (see also my comments below).

Reply:
The scope statement has been removed, and `use` and `borrow` take optional blocks.